vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+# Reconcile Mouse APIs in React Native - Desktop
+
+Windows and macOS support mouse devices today, but their React Native counterparts are not fully flushed out yet to unblock customers attempting to create React Native apps on those devices.
+
+This document will attempt to capture what's needed within React Native for Windows and macOS to enable customer scenarios. This includes:
+
+- Aligning with the [react-native-web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0) mouse behavior/implementation [issue TBD]()
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
+Property on `Pressable` that determines whether or not the mouse is currently within the bounds of the designated content.
+
+| TYPE | REQUIRED | PLATFORM |
+|:--:|:--|:--|
+| boolean | Yes | Windows, macOS |
+
+[React Native Web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0#diff-9fe3ab194c72e9837a8ffc3e7c44f33d) introduced  `hovered` on `Pressable` to enable common mouse scenarios. In order to align and make this complete the work is as follows:
+
+- Windows has added `hovered` and marked `onMouseEnter/Leave` as to be deprecated [issue TBD]()
```

Comment:
Interested in some more details on this and the below macOS issue.

`onMouseEnter/onMouseLeave` exist on View/Touchable right now but we've actually left it off of `Pressable`. Would this be a noop, or would there be a desire to remove the prop from View/Touchable?

Our partners have built things with `onMouseEnter/onMouseLeave` that cannot be accomplished with hover state in the style callback, so I'm wondering if we should wait until we have more generalized mouse events beyond this until we try tro deprecate.